dashboard/pipeline.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
same as the file before for here either do all of them as from os import x,y,z  or just import the entire thing